### PR TITLE
Add optional serial number to file

### PR DESCRIPTION
**Pull request by xvilo**
This will give one the ability to add `{serial}` to the URL. This will be replaced with the PI's serial number

### CODE REVIEW

File: src/modules/fullpageos/filesystem/home/pi/scripts/start_chromium_browser [hunk 1]
```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 
+SERIAL=`cat /proc/cpuinfo | perl -n -e '/^Serial[ ]*: ([0-9a-f]{16})$/ && print "$1\n"'`
+
 # Standard behavior - runs chrome
-chromium-browser --kiosk --touch-events=enabled --disable-pinch --noerrdialogs --disable-session-crashed-bubble --app=$(head -n 1 /boot/fullpageos.txt)
+chromium-browser --kiosk --touch-events=enabled --disable-pinch --noerrdialogs --disable-session-crashed-bubble --app=$(head -n 1 /boot/fullpageos.txt | sed -e "s/{serial}/${MYVAR}/g")
```
Comment (dbursem) [inline src/modules/fullpageos/filesystem/home/pi/scripts/start_chromium_browser]: shouldn't you replace with ${SERIAL} instead of ${MYVAR} ?

Comment (xvilo) [reply src/modules/fullpageos/filesystem/home/pi/scripts/start_chromium_browser]: Oops! That was a wrong copy-past from my test. Fixed it now

**Comment by guysoft** [comment]
Fixes #251

**Comment by xvilo** [comment]
@guysoft for future refrence, just add another variable with data you want and add `-e "s/search-regex/${NEW_VAR}/g` to current sed command.

**Comment by xvilo** [comment]
Yea, I had issues too. Didn't had the time to fix, investigate and report it :P

**Comment by guysoft** [comment]
@lippoliv Can you make a PR please.

**Comment by guysoft** [comment]
Also this should be documented in the wiki

**Pull request merged**